gen_prejit.sh

```
@@ -0,0 +1,183 @@
+echo "[Step 1] Reminder: if you added a new LAGraph algorithm, make sure its"
+echo "         test is included in LAGRAPH_FALKORDB_TESTS in build.sh."
+echo "         Press Enter to continue or Ctrl-C to abort and update it first."
+read -r
+
+# ---------------------------------------------------------------------------
+# Step 2 – Remove the previous build output so the warm-up is done from
+#           scratch (prevents stale kernels from being reused instead of
+#           regenerated).
+# ---------------------------------------------------------------------------
+echo "[Step 2] Removing previous build output (bin/)..."
+cd "${FALKORDB_DIR}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)